### PR TITLE
Remove some constants from features.go

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@verult Please take a look at this PR. I have removed an uncommon practice in kubernetes repo with this commit.

File: pkg/features/features.go [hunk 1]
```diff
@@ -19,8 +19,6 @@ package features
 import utilfeature "k8s.io/apiserver/pkg/util/feature"
 
 const (
-	// owner: @verult
```
Comment (msau42) [inline pkg/features/features.go]: I think we should keep this so we know who are the feature owners like in k/k

Comment (humblec) [reply pkg/features/features.go]: @msau42 dont we record it in `OWNERS` file instead of individual source file? I thought thats the general practice.

Comment (davidz627) [reply pkg/features/features.go]: It's not a file-by-file ownership as much as a feature ownership that can span several files/packages

Comment (humblec) [reply pkg/features/features.go]: @davidz627 not sure I got your last comment . Are you saying invidual file mention  of owner is fair enough ?

Comment (davidz627) [reply pkg/features/features.go]: What I'm saying is
`OWNERS` file -> specifies ownership over some file or folders (related files)
`features.go` comments -> specifies ownership over some feature that may span several unrelated files, and does not imply ownership over the files in general

**Comment by humblec** [comment]
@msau42 @davidz627 thanks. Closing this PR based on the review!

**Pull request closed without merging**